### PR TITLE
Updated getting-started.md

### PR DESCRIPTION
**Pull request by TheDutchCoder**
Small changes to conclusion for better flow into next guide.

### CODE REVIEW

File: content/guides/getting-started.md [hunk 1]
```diff
@@ -257,7 +257,8 @@ T> Custom parameters can be passed to webpack by adding two dashes between the `
 
 ## Conclusion
 
-Now that you have a basic build together, you should dig into the [basic concepts](/concepts) and [configuration](/configuration) to better understand webpack's design. The [API](/api) section digs into the various interfaces webpack offers. Or, if you'd prefer to learn by example, select the next guide from the list and continue building out this little demo we've been working on which should now look similar to this:
+Now that you have a basic build together you should move on to the next guide [`Asset Management`](/guides/asset-management) to learn about how to manage assets like images and fonts with webpack. By now, your project should look like this:
+
```
Comment (skipjack) [inline content/guides/getting-started.md]: Drop the extra line.

File: content/guides/getting-started.md [hunk 2]
```diff
@@ -257,7 +257,8 @@ T> Custom parameters can be passed to webpack by adding two dashes between the `
 
 ## Conclusion
 
-Now that you have a basic build together, you should dig into the [basic concepts](/concepts) and [configuration](/configuration) to better understand webpack's design. The [API](/api) section digs into the various interfaces webpack offers. Or, if you'd prefer to learn by example, select the next guide from the list and continue building out this little demo we've been working on which should now look similar to this:
+Now that you have a basic build together you should move on to the next guide [`Asset Management`](/guides/asset-management) to learn about how to manage assets like images and fonts with webpack. By now, your project should look like this:
```
Comment (skipjack) [inline content/guides/getting-started.md]: s/learn about how/learn how

Comment (skipjack) [inline content/guides/getting-started.md]: I might rephrase `By now, your project should look like this:` to something like `At this point, the project should look something like this:`.

Comment (TheDutchCoder) [inline content/guides/getting-started.md]: 👍

**Pull request merged**